apps/skateboard/web/channels/game_channel.ex

```diff
@@ -0,0 +1,9 @@
+defmodule Skateboard.GameChannel do
+  use Phoenix.Channel
+
+  def join("game:new", _message, socket) do
+    id = socket.assigns.user_id
+    user = Skateboard.Repo.get(User, id)
```

Comment:
This is where you would add the player to the elixir game and send the game state to the client